packages/mc-scripts/config/create-webpack-config-for-production.js

```diff
@@ -120,7 +78,10 @@ module.exports = ({
     // https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a
     optimization: {
       minimizer: [
-        new UglifyJsPlugin(uglifyConfig),
+        new TerserPlugin({
```

Comment:
woah! this is too simple 👍  😍

Reply:
It has some options too but these are the most useful I think. The compression options we can opt-into once we need them I guess. The branch builds fine. Will dig a bit into size comparison before merging.

Reply:
Sizes are very similar. Nothing to hold back the migration I think.